### DISCUSSION TITLE
No way to create PR

### DISCUSSION THREAD

**Discussion by miklyx** [DevSkillsHQ/discusssions#69] [Help] [answered]
Hi! 
What about impossibility of making PR?

`There isn’t anything to compare.
main and implementation are entirely different commit histories.`

I have [implementation](https://github.com/miklyx/data4-life-meal-roulette-app-mobile-h50jf-bh9ic0/tree/implementation) branch and [from_scratch](https://github.com/miklyx/data4-life-meal-roulette-app-mobile-h50jf-bh9ic0/tree/from_scratch) branch (I've created it because of boilerplate does not work) in my repo, and there's no way to make PR.

Now I've pushed code from `from_scratch` to `implementation`.

And I don't have any ways to create PR (see message above).

**Comment by dgikonyo** [comment]
Check on how to cloned your project. Ideally you ought to have cloned from the "master" branch.
This is how I did mine.

```bash 
git clone <github url>
git checkout -b implementation
```

Try that and you can reach out.

**Reply by miklyx** [reply]
Thanks for the answer. 
I did exactly the same.
Now my code exists in `implementation` branch, but I can not make pull request to `main` branch of original repo - that was my question.

**Reply by fenske** [reply, marked as answer]
hi @miklyx. I'm Anton from Alva. From what I can tell, you created a fork from the original challenge repository. But the idea was for you to just create a new branch in the original challenge repo as you have write access to it.

Here's how I'd resolve this:

1. Clone the challenge repo (or create a GitHub Codespace): https://github.com/ffc1e12/data4-life-meal-roulette-app-mobile-h50jf-bh9ic0
2. Copy and paste the contents of the [implementation branch of your forked repo](https://github.com/miklyx/data4-life-meal-roulette-app-mobile-h50jf-bh9ic0/tree/implementation) into the cloned repo above.
3. Create a new branch on the cloned repo, commit it and push.
4. Create a PR.

Feel free to let me know if you need help performing these steps.

BR,
Anton.

**Reply by miklyx** [reply]
@fenske  thank you, this is kinda works.
